releasenotes/notes/cx-cancellation-pass-generalization-538fb7cfe49b3fd5.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    Introduced a new feature ``Cancellation`` that generalizes the ``CXCancellation`` 
+    pass to cancel any self-inverse gates or gate-inverse pairs. It can be used by 
+    initializing ``Cancellation`` and passing a gate to cancel, for example::
+
+      from qiskit.transpiler.passes import Cancellation
+      from qiskit.circuit.library import HGate
+      Cancellation([HGate])
```

Comment:
This is good! Since it's almost exclusively going to be used with the PassManager, could you include that as well? It can be just like in the tests:
```
pass_ = Cancellation([HGate()])
pm = PassManager(pass_)
new_circ = pm.run(qc)
```